setup.cfg

```diff
@@ -44,6 +44,7 @@ all =
     openslide-python==1.1.2
     pandas
     einops
+    transformers==4.10.2
```

Comment:
I think we can try to use the latest version for now, as we have the nightly test to detect any compatibility issues

Reply:
Sure, reflected